src/System.IO.Ports/src/Interop/Unix/Interop.Termios.cs

```diff
@@ -10,13 +11,17 @@ internal static partial class Interop
 {
     internal static partial class Termios
     {
+        [Flags]
         internal enum Signals
         {
-            SignalDtr = 1,
-            SignalDsr = 2,
-            SignalRts = 3,
-            SignalCts = 4,
-            SignalDcd = 5,
+            None = 0,
+            SignalDtr = 0b000001,
+            SignalDsr = 0b000010,
+            SignalRts = 0b000100,
+            SignalCts = 0b001000,
+            SignalDcd = 0b010000,
+            SignalRng = 0b100000,
```

Comment:
You don't want to define these with the same syntax as you do in the native equivalents? e.g.
```C#
...
SignalDsr = 1 << 1,
...
```
?